spec/system/order_system_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Order', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  describe 'Points' do
```

Comment:
I suggest something a bit more descriptive
```suggestion
  describe 'Ordering Rewards' do
```